openedx/core/djangoapps/content_libraries/tests/test_runtime.py

```diff
@@ -169,8 +168,7 @@ def test_xblock_metadata(self):
         """
         Test the XBlock metadata API
         """
-        unit_block_key = library_api.create_library_block(self.library.key, "unit", "metadata-u1").usage_key
-        problem_key = library_api.create_library_block_child(unit_block_key, "problem", "metadata-p1").usage_key
```

Comment:
There's no `create_library_block_child` method in the api anymore -- and LearningCoreXBlockRuntime doesn't support child blocks, so I've removed parent-child relationships from these tests.